app/src/main/kotlin/rasel/lunar/launcher/todos/TodoManager.kt

```diff
@@ -57,6 +57,12 @@ internal class TodoManager : Fragment() {
     override fun onResume() {
         super.onResume()
         refreshList()
+        lActivity!!.viewPager.isUserInputEnabled = false
```

Comment:
doesn't this disallow to scroll the to-do list?

Reply:
I don't think it should effect the list in the todo manager.
It's supposed to disable the swipeability of the viewPager while the todo manager is open. And it should re-enable it when the todo manager closes ( `onPause()` )